### PR TITLE
Added restaurants and activities to iamdv

### PR DESCRIPTION
**Pull request by iamdv**
:wave: I added some restaurants in London.

:bulb:

### CODE REVIEW

File: iamdv-london.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# iamdv based in London
+
+### Restaurants
+
+ - __bold__ Vapiano near London Bridge __bold__
```
Comment (jurajsucik) [inline iamdv-london.md:5]: @iamdv what kind of food do they serve?

Comment (iamdv) [reply iamdv-london.md:5]: @jurajsucik Italian food by Germans 😉

File: iamdv-london.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+ - __bold__ Wahaca near Southbank __bold__
```
Comment (jurajsucik) [inline iamdv-london.md:7]: @iamdv Cool! anything else?

**Review by jurajsucik — APPROVED** [review]
Looks good

**Pull request merged**